scripts/prep_debug_env.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+DEBUG_PORT=$1
+PROJECT=${ASB_PROJECT:="ansible-service-broker"}
+
+oc set probe dc/asb --remove --readiness --liveness -n ${PROJECT}
```

Comment:
We have the https://github.com/automationbroker/automation-broker-apb maybe this is something that could be added there (ie. you provision a debuggable broker).

Reply:
Sure I can take a look there also